### ISSUE TITLE
VS integration test hang waiting for solution crawler

### ISSUE THREAD

**Issue by dpoeschl** [dotnet/roslyn#21125]
https://ci.dot.net/job/dotnet_roslyn/view/Official%20Builds/job/master/job/windows_debug_vs-integration_prtest/6721/artifact/Binaries/Debug/

Dumps backed up to \\mlangfs1\public\davip\dumps\jenkins\vsintegration\hangwaitingforsolutioncrawler

**Comment by dpoeschl** [comment]
Looking to see what test it is

**Comment by Pilchie** [comment]
@heejaechang any ideas?

**Comment by heejaechang** [comment]
looking. it can happen if one of analyzer doesn't return.

**Comment by heejaechang** [comment]
unless those code are changed, things that are awaiting (except low priority queue) don't run user code and has been running for years. do you know how long it was sitting like that? any one knows how to see what Task is pending in VS?

**Comment by heejaechang** [comment]
do we have a dump for service hub process as well? (ServiceHub.RoslynCodeAnalysisService32.exe) I suspect that this wasn't returning (http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/SolutionCrawler/WorkCoordinator.NormalPriorityProcessor.cs,476) but without dump of OOP, can't be sure. 

since OOP is set to below normal priority, it could taking time to return to respond... not sure without OOP dump at the time.

**Comment by dpoeschl** [comment]
> do we have a dump for service hub process as well?

Nope, but we should... I don't know if there's an include list of processes to dump or what, but I can take a look.
